src/utils/link-interceptor.js

```diff
@@ -25,6 +25,11 @@ export const interceptLinks = event => {
     return true;
   }
 
+  const newTab = event.ctrlKey || event.metaKey;
+  if (newTab) {
+    return true; // will open in new tab b/c we dont eat the event
```

Comment:
if it's an `<a>` it will cause the browser to respond differently.

probably not on a `<div>` but I do like to keep the pattern intact.